.changeset/nervous-mails-heal.md

```diff
@@ -0,0 +1,7 @@
+---
+"@yamada-ui/autocomplete": patch
+"@yamada-ui/select": patch
+"@yamada-ui/use-popper": patch
```

Comment:
This line might be for `@yamada-ui/popover` !

Reply:
Fixed.